src/main/resources/locales/zh-HK.yml

```diff
@@ -117,122 +195,305 @@ commands:
+      success-one: '&a所有島嶼的 “&b[name]&a” 保護標志已重置為默認值。'
     world:
       description: "管理世界設置"
     delete:
-      parameters: "<player>"
+      parameters: <player>
       description: "刪除玩家的島嶼"
       cannot-delete-owner: "&c刪除之前必須將所有島嶼成員都踢出島嶼。"
       deleted-island: "&a位於 &e[xyz] &a的島嶼已經被成功刪除。"
+    deletehomes:
+      parameters: <player>
+      description: '從島上刪除所有已命名的Home點'
+      warning: '&c島上所有已命名Home點將被刪除！'
     why:
-      parameters: "<player>"
+      parameters: <player>
       description: "切換控制台保護調試報告"
       turning-on: "打開 [name] 的控制台調試信息"
       turning-off: "關閉 [name] 的控制台調試信息"
     deaths:
       description: "強制設置玩家的死亡次數"
       reset:
         description: "強制重置玩家的死亡次數為零"
-        parameters: "<player>"
+        parameters: <player>
+        success: '&a成功重置玩家 &b[name] &a的死亡次數。'
       set:
         description: "強制設置玩家的死亡次數"
-        parameters: "<player> <deaths>"
+        parameters: <player> <deaths>
+        success: '&a成功將玩家 &b[name] &a的死亡次數設置為 &b[number] &a次。'
+      add:
+        description: '增加玩家的死亡次數'
+        parameters: <player> <deaths>
+        success: |-
+          &a成功將玩家 &b[name] &a的死亡次數增加 &b[number] 次。
+          &a他現在的死亡次數為： &b[total] &a次。"
+      remove:
+        description: '減少玩家的死亡次數'
+        parameters: <player> <deaths>
+        success: |-
+          &a成功將玩家 &b[name] &a的死亡次數減少 &b[number] 次。
+          &a他現在的死亡次數為： &b[total] &a次。"
   bentobox:
-    description: "BentoBox 管理員命令"
+    description: 'BentoBox 管理員命令'
     about:
-      description: "顯示版權和協議信息"
+      description: '顯示版權和協議信息'
     reload:
-      description: "重新載入所有語言文件"
+      description: '重載 BentoBox 和所有附加組件、設置和語言'
       locales-reloaded: "&2語言文件已經重新載入"
       addons-reloaded: "&2組件已經重新載入"
+      settings-reloaded: '[prefix_bentobox]&2設置已重載。'
+      addon: '[prefix_bentobox]&6正在重載 &b[name] &2。'
+      addon-reloaded: '[prefix_bentobox]&b[name] &2已重載。'
+      warning: '[prefix_bentobox]&c警告： 重載可能導致不穩定， 如果發生錯誤， 請重啟服務器。'
+      unknown-addon: '[prefix_bentobox]&c未知組件！'
+      locales:
+        description: '重載語言資源(zh-HK)'
     version:
       plugin-version: "&2Bentobox 版本：&3[version]"
-      description: "顯示信息"
-      loaded-addons: "已加載組件"
-      loaded-game-worlds: "已加載遊戲世界："
-      addon-syntax: "&2[name] &3[version] &7(&3[state]&7)"
-      game-world: "&2[name] &7(&3[addon]&7): &a Overworld&7, &r Nether&7, &r End"
+      description: '顯示 BentoBox 和附加組件版本'
+      loaded-addons: '已載入的附加組件：'
+      loaded-game-worlds: '已載入的游戲世界：'
+      addon-syntax: '&2[name] &3[version] &7(&3[state]&7)'
+      game-world: '&2[name] &7(&3[addon]&7) : &3[worlds]'
+      server: '&2服務器： &3[name] [version]&2'
+      database: '&2數據庫： &3[database]'
+    manage:
+      description: '顯示管理面板'
+    catalog:
+      description: '顯示編目'
+    locale:
+      description: '執行語言文件分析'
+      see-console: |-
+        [prefix_bentobox] &a請從控制台讀取反饋信息。
+        [prefix_bentobox] &a此命令非常垃圾，無法從聊天中讀取反饋...
+        [prefix_bentobox] &a確實垃圾，就算從控制台裡讀取反饋也非常吃力啊！
```

Comment:
Is this line required? File a bug report if you think this and we'll try to improve it!

Reply:
Actually, if I may, I would like to suggest some improvements regarding this part.

1) Adding an extra parameter to command "/bentobox locale"
when using the command "/bentobox locale", adding an extra parameter [all/active] would be useful. "All" means detect discrepancies for all languages while "active" means only detect for the language currently in use. The rationale is that other languages that I am not using is not useful to have them checked for any discrepancy between that and default language. Another reason is that currently most of the locale files are not up-to-date which results in a very spammy console, considering some addons may be installed as well.
2) Export to text file
When there are multiple addons installed, the console would become extremely spammy and hard to extract or trace any useful data. An option to export the result into a file or simply direct extraction would be useful. If it is to be extracted, the console could be just showing how many files had been detected with discrepancy(ies) found.

Feel free to let me know if you need any further elaborations on the above.
Happy Lantern Festival!